build-system/pr-check/module-tests.js

```diff
@@ -26,14 +26,26 @@ const {
   timedExecOrDie,
 } = require('./utils');
 const {buildTargetsInclude, Targets} = require('./build-targets');
+const {MINIFIED_TARGETS} = require('../tasks/helpers');
 const {runCiJob} = require('./ci-job');
 
 const jobName = 'module-tests.js';
 
+function prependConfig() {
+  for (const target of MINIFIED_TARGETS) {
+    for (const ext of ['js', 'mjs']) {
+      timedExecOrDie(
+        `gulp prepend-global --${process.env.config} --local_dev --fortesting --derandomize --target=dist/${target}.${ext}`
+      );
+    }
+  }
+}
```

Comment:
I just thought of something that will make this much simpler. If you add a new `targets` flag to `prepend-global` that applies the config to a list of files, you won't need this loop or the one in `nomodule-tests.js`.

Right now, the logs are filled with this:
![image](https://user-images.githubusercontent.com/26553114/109055003-a20ac780-76ac-11eb-8c15-daef9761919b.png)

Reply:
I'd rather contain this PR to just this one change...

Reply:
You're already changing `prepend-global/index.js` in this PR, so I think it's within scope here.

Reply:
It's a much larger change that requires separate testing - I'll add a TODO